pom.xml

```diff
@@ -72,43 +68,40 @@
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
-    <maven-hpi-plugin.version>1.96</maven-hpi-plugin.version>
+    <jenkins.version>1.609.3</jenkins.version>
```

Comment:
I bumped this back when I thought the best version to go to was 1.609.3.  I changed it to 1.580.1 since 1.520 has some issues with the test harness.